src/components/formreporttype.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
+import * as Yup from "yup";
+import ErrorMsg from "./errormsg";
+
+const getSubCategories = (categories, categoryId) => {
+  var cats = categories.find(category => category.id === categoryId);
+  return cats ? cats.types : [];
+};
+
+const ServiceRequestForm = props => {
+  const [categories] = useState(Categories);
+  const [subCategories, setSubCategories] = useState([]);
+
+  const handleServiceRequestChange = changeEvent => {
+    const { value } = changeEvent.currentTarget;
+    const subCategories = getSubCategories(categories, parseInt(value));
+    setSubCategories(subCategories);
+  };
+
+  return (
+    <React.Fragment>
+      <div className="container Container-bg">
+        <h4>How Can We Help?</h4>
+        <Formik
+          initialValues={{
+            requestType: "",
+            subRequestType: ""
+          }}
+          validationSchema={Yup.object().shape({
+            requestType: Yup.string().required("Request Category is required"),
+            subRequestType: Yup.string().required("Sub Category is required")
+          })}
+          onSubmit={(values, { setSubmitting }) => {
+            setTimeout(() => {
```

Comment:
i'm not sure you need this here, I htink maybe its just similating you posting to the server and that taking some time, please rmeove it and just leave your alert and set statement.

Reply:
i'm assuming this came from the demo?

Reply:
removed timer function